_includes/sidenav.html

```diff
@@ -7,7 +8,7 @@
         {% assign _current = true %}
       {% endif %}
     <li class="usa-sidenav__item">
-      <a href="{% if link.external == true %}{{ link.href }}{% else %}{{ link.href | relative_url }}{% endif %}"
+      <a href="{% if link.external == true %}{{ link.href }}{% else %}{{ page_url }}{{ link.href | relative_url }}{% endif %}"
```

Comment:
Prior to this change, the links were appearing relative to the root (http://domain/#anchor) when it should have been something like http://domain/page.url/#anchor.  I couldn't just add `page.url` to the anchor tag line b/c it has a trailing slash, which would cause the URL to be http://domain//#anchor.

I think the `minus:` filter would simplify this - I'll test shortly.

Reply:
Nope - minus: doesn't do it, looks like we could create a custom filter, like shopify's `slice`.  But this will do for now IMO.

Reply:
I suppose we could remove the `relative_url` filter; I can't think of how that would be a problem, since the link is not dynamically calculated and shouldn't be subject to some kind of attack...

Reply:
what about `link.href | prepend: site.baseurl` ?

Reply:
I think simply `{{ link.href }}` will do because it generates `href="#<anchorname>"`, which the browser will naturally make relative to the page.